### ISSUE TITLE
Improve LibGetFrame integration

### ISSUE THREAD

**Issue by mrbuds** [WeakAuras/WeakAuras2#4622]
**Is your feature request related to a problem? Please describe.**

External glow a unit frame, Anchor an aura to a unit frame and Dynamic Group's grow to unit frame, all use default LibGetFrame settings

Sometime a user wants to change priority, ignore some frame, or add one and they need to use a custom function instead which is complicated for average user.

**Describe the solution you'd like**
2 options:
- implement these options in wa
- implement a mini panel of options in lib, wa options have a button to call it and it return a table, kinda like with colorpicker

**Comment by InfusOnWoW** [comment]
I think a global options for that is saner. An aura author mostly wants to set it up so that the user's unit frame gets anchored, glowed, etc.

And the user needs to decide what their "unit frame" is, if they have multiple installed. 

I think @Stanzilla had some half-baked general settings screen?

**Comment by Stanzilla** [comment]
Yeah it is in https://github.com/WeakAuras/WeakAuras2/tree/default-options-for-real
